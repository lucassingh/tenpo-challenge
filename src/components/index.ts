export * from './UI/Layout'
export * from './UI/Loader'
export * from './UI/Header'
export * from './UI/UserNav'
export * from './UI/CTA'
export * from './UI/MainLayout'
export * from './UI/SubHeader'
export * from './UI/PrimaryButton'
export * from './home/SliderRestaurant'
export * from './home/SliderCategories'
export * from './home/SliderFavorites'
export * from './search/Map'
export * from './search/HeaderSearch'
export * from './search/SearchBar'
