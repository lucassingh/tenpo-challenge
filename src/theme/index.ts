import light from './lightTheme';

export default { light }