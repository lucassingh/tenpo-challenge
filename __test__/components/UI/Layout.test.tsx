import React from "react";
import { Layout } from "../../../src/components";
import { render } from '@testing-library/react-native'

describe('Test Layout component <Layout />', () => {
    
    it('Should match with snapshot', () => {

        const container  = render(<Layout />);

        expect(container).toMatchSnapshot()
    })
})
