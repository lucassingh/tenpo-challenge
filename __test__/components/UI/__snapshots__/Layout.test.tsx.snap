// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`Test Layout component <Main /> Should match with snapshot 1`] = `
<View
  style={
    [
      {
        "backgroundColor": "#F2F2F2",
        "center": "",
        "display": "flex",
        "flexBasis": 0,
        "flexGrow": 1,
        "flexShrink": 1,
        "justifyContent": "center,align-items:",
      },
    ]
  }
/>
`;
